matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/model/TimelineEventEntity.kt

```diff
@@ -46,7 +46,5 @@ internal fun TimelineEventEntity.deleteOnCascade(canDeleteRoot: Boolean) {
     if (canDeleteRoot) {
         root?.deleteFromRealm()
     }
-    annotations?.deleteOnCascade()
-    readReceipts?.deleteOnCascade()
```

Comment:
Not needed anymore? Can you explain?

Reply:
Annotations and read receipts persistence is supposed to be independent
 from timeline. This is what caused reactions to be lost during gap

Reply:
So the table for annotations will grow up indefinitely, right?

Reply:
Yes until cache is cleared